src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -213,6 +196,23 @@ final class HTTPClient: Client {
 		}
 	}
 
+	func exposureConfiguration(
+		completion: @escaping ExposureConfigurationCompletionHandler
+	) {
+		log(message: "Fetching exposureConfiguration from: \(configuration.configurationURL)")
+		appConfiguration { config in
+			guard let config = config else {
+				completion(nil)
+				return
+			}
+			guard config.hasExposureConfig else {
+				completion(nil)
+				return
+			}
+			completion(try? ENExposureConfiguration(from: config.exposureConfig, minRiskScore: config.minRiskScore))
```

Comment:
the docs in the protocol say the completion handler will always be called on the main thread. is this guaranteed here?

Reply:
This comment is wrong. I removed it.
This function is only used by the ExposureDetection.
The ExposureDetection "forwards" this call to the UI on the main thread.